workshops/migrations/0060_published_help_text.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+import re
+import django_countries.fields
+
+
+def add_address_to_online_events(apps, schema_editor):
+    """Set Event.address if empty for online events."""
+    Event = apps.get_model('workshops', 'Event')
+    Tag = apps.get_model('workshops', 'Tag')
+    online = Tag.objects.get(name='online')  # should be created via 0040_*
+
+    Event.objects.filter(tags__in=[online]) \
+                 .filter(models.Q(address=None) | models.Q(address='')) \
+                 .update(address='Internet')
```

Comment:
The intent of this migration was to fill up `venue` for all online events, since the `0040` migration didn't set it.

I disagree about the DRY – we keep tags for to easily show the type of the event (DC / SWC / LC / online / TTT / stalled [coming soon]).

I don't want to put another constraint on the admins (event has 'online' tag? Therefore it's location must be in 'Internet'!) because there can be instances of events that may have a different location (like streamed events?).

The `0040` migration did set `venue` because I couldn't think of a good value for that field (I still can't - I reuse 'Internet' for `address` here).

I agree that this migration should also set the `address` for non-empty addresses for events with W3 country. I'll update shortly.